mod-data-export/mod-data-export.postman_collection.json

```diff
@@ -1773,7 +2223,8 @@
 					"                    \"data-export.file-definitions.item.get\",",
 					"                    \"data-export.file-definitions.item.post\",",
 					"                    \"data-export.file-definitions.upload.post\",",
-					"                    \"data-export.job-executions.items.get\"",
+					"                    \"data-export.job-executions.items.get\",",
+					"                    \"inventory-storage.all\"",
```

Comment:
It is better to restrict the permissions to just those that are needed. That way it will help us determine any missing permission issues.

Reply:
Also why does the regular user need to have inventory permission?
This user just needs to have data-export permissions. All the pre-requisite data can be created using the admin user